contribs/drt/src/main/java/org/matsim/contrib/drt/optimizer/Waypoint.java

```diff
@@ -52,12 +52,14 @@ class Start implements Waypoint {
 		public final Link link;
 		public final double time;
 		public final int occupancy;
+		public final double now;
```

Comment:
Maybe `createTime` is less confusing than `now`? Maybe we could have `createTime` even at the `VehicleEntry` level?

Why not pass the current time to `InsertionDetourTimeCalculator`?

Reply:
Yes, `createTime` sounds good to me, I didn't put `now` to `VehicleEntry` because there it was even less clear what it means. But putting `createTime` into `VehicleEntry` is perfect, then it means "when the entry was created".

My initial attempt was to pass the time over to `InsertionDetourTimeCalculator`, but this created a whole chain of passing it through the whole infrastructure, over the Extensive/Selective classes etc., which exploded the number of changed files + the unit tests needed to be adapted. So adding it to VehicleEntry/Start looks clearner to me.